src/pages/[platform]/build-a-backend/data/subscribe-data/index.mdx

```diff
@@ -81,6 +81,102 @@ export default function MyComponent() {
 
 `observeQuery` fetches and paginates through all of your available data in the cloud. While data is syncing from the cloud, snapshots will contain all of the items synced so far and an `isSynced` status of `false`. When the sync process is complete, a snapshot will be emitted with all the records in the local store and an `isSynced` status of `true`.
 
+<Accordion title='Missing real-time events and model fields' headingLevel='4' eyebrow='Troubleshooting'>
```

Comment:
should this be heading level 3?

Reply:
You tell me! I copied another `Accordion` and it looked OK to me. I don't see what changing the level actually does. 😅